README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
+for additional information.
+
+### Adjusting global server settings
+In tandem with the `mscs.properties` file--which handles options for individual 
+worlds--
+is the `mscs.conf` or simply `mscs` file, which handles global server settings.
+
+**You must create this file**. By default, MSCS looks for this file in three 
+places (in this order):
+
+1. `$HOME/mscs.conf` 
+
+2. `$HOME/.config/mscs/mscs.conf`
+
+3. `/etc/default/mscs`-- **if you save it in this location it is only called 
+`mscs`, NOT `mscs.conf`**
+
+It doesn't matter where you create the file, as long as you put it in one of 
+the above places.
+
+Please note: `$HOME` represents the home directory of the user that is 
+responsible for the script--
+if you followed the configuration above, then that would be the `minecraft` 
+user. 
+To print the home directory of the `minecraft` user type:
+
+`eval echo ~$minecraft`
+
+Then simply `cd` into the directory it outputs and create the file.
+
+Once you've created the file, you need to populate it with a list of properties. 
+The list of properties can be found 
+[here](https://github.com/sandain/MinecraftServerControlScript/wiki/Global-Server-Settings).
+
+## Automated backups and restarts
+
+### Scheduling backups
+MSCS uses [**cron**](https://en.wikipedia.org/wiki/Cron), 
+a scheduler software that can run programs on a set interval of time. 
+Below is an example of one way how you could setup backups via cron to backup 
+a world every 2 hours:
+
+Type the following (in any directory): 
+
+  ````
+  export EDITOR=vim
+  crontab -e
+  ````
+  Page down until you get to an empty line. Then paste the following:
+  
+  ````
+  0 */2 * * *  /usr/local/bin/mscs backup myWorld
+  ````
+  * `0 */2 * * *` is the time interval to backup. 
+This particular expression means backup every 2 hours. 
+You can change this to 3, 4, 5 or whatever amount of hours to backup 
+X amount of hours. 
+You can also backup according to days, minutes, seconds, the time of 
+the day, and more. 
+See [the wiki page]
+(https://github.com/sandain/MinecraftServerControlScript/wiki/Backup-and-Restore) 
+for more information.
+  * `myWorld` is the name of the world you wish to backup. 
+  Omitting this will backup all worlds.
+
+  Finally, press escape, then type
+  `:wq`
+  to save and quit.
+  
+The backups will be saved in `/opt/mscs/backups`. 
+  
+### Removing backups after X days
+You can specify how long to keep backups by changing the `BACKUP_DURATION` 
+in the `mscs.conf` or `mscs` file 
+(see [adjusting global server settings](#adjusting-global-server-settings)).
+
+### Scheduling restarts
+You can schedule restarts for the server following the same method as outlined 
+in [scheduling backups](#scheduling-backups).  
+Simply change the scheduled command from `backup <myWorld>` to 
+`restart <myWorld>`. `myWorld` 
+is the name of world you wish to restart; omit if you wish to 
+restart all worlds.
+
+## Mapping the world
```

Comment:
This can be added in the future, but this section should mention `overviewer-settings.py`
By default, mscs creates this file with defaults but it can be further customized for each server.